Doc/glossary.rst

```diff
@@ -383,7 +385,7 @@ Glossary
 
    file object
       An object exposing a file-oriented API (with methods such as
-      :meth:`read()` or :meth:`write()`) to an underlying resource.  Depending
+      :meth:`!read` or :meth:`!write`) to an underlying resource.  Depending
```

Comment:
It can refer to `io.BaseIO`.

Reply:
I'm not sure there's anything we could specifically link to for either method. `io.IOBase` does not have `read()` or `write()` as abstract methods, so neither `IOBase.read` nor `IOBase.write` is explicitly documented currently.

https://docs.python.org/3/library/io.html#io.IOBase

Here's all the `IOBase` docs say on `read` and `write`:

> Even though [IOBase](https://docs.python.org/3/library/io.html#io.IOBase) does not declare read() or write() because their signatures will vary, implementations and clients should consider those methods part of the interface. Also, implementations may raise a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) (or [UnsupportedOperation](https://docs.python.org/3/library/io.html#io.UnsupportedOperation)) when operations they do not support are called.

Specific subclasses of `IOBase` implement versions of `read` and `write` that either only work with raw binary streams or only work with text streams

Reply:
Well, and there is a reason to not declare them there. But there are many other references to `read` and `write`. It would be nice to have the general interface of them be documented somewhere. It is a different issue.